tests/jobs/test_scheduler_job.py

```diff
@@ -2367,6 +2370,194 @@ def test_runs_are_created_after_max_active_runs_was_reached(self, dag_maker, ses
         dag_runs = DagRun.find(dag_id=dag.dag_id, session=session)
         assert len(dag_runs) == 2
 
+    @pytest.mark.parametrize(
+        "ti_state, final_ti_span_status",
+        [(State.SUCCESS, SpanStatus.ENDED), (State.RUNNING, SpanStatus.ACTIVE)],
+    )
```

Comment:
pytest.param lets us give pretty names to parameterized tests, the `id` will get appended to the test name for easier debugging if one fails.

```suggestion
    @pytest.mark.parametrize(
        "ti_state, final_ti_span_status",
        [
            pytest,param(State.SUCCESS, SpanStatus.ENDED, id="ended_successfully"),
            pytest.param(State.RUNNING, SpanStatus.ACTIVE, id="still_active")
        ],
    )
```

Reply:
Good to know! Done.